### PR TITLE
docs: add Notes

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: add Notes |
| 🇨🇳 Chinese | docs: add Notes |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/anchor/index.en-US.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Hyperlinks to scroll on one page.
 
 For displaying anchor hyperlinks on page and jumping between them.
 
+> Notes for developers
+>
+> After version `4.24.0`, we rewrite Anchor use FC, Some methods of obtaining `ref` and calling internal instance methods will invalid.
```
Comment (afc163) [inline components/anchor/index.en-US.md:17]: changelog 里也需要体现一下。

Comment (li-jia-nan) [reply components/anchor/index.en-US.md:17]: changelog 是谁负责写呢？

Comment (afc163) [reply components/anchor/index.en-US.md:17]: @xrkffgg

**Review by afc163 — APPROVED** [review]

File: components/space/__tests__/__snapshots__/demo-extend.test.ts.snap [hunk 2]
```diff
@@ -12709,7 +12709,14 @@ Array [
...
       class="ant-switch-inner"
-    />
+    >
+      <span
+        class="ant-switch-inner-checked"
+      />
+      <span
+        class="ant-switch-inner-unchecked"
+      />
+    </span>
```
Comment (afc163) [inline components/space/__tests__/__snapshots__/demo-extend.test.ts.snap:12719]: ci 挂了

Comment (li-jia-nan) [reply components/space/__tests__/__snapshots__/demo-extend.test.ts.snap:12719]: 这个快照我在本地跑是没问题的，奇怪

Comment (MadCcc) [reply components/space/__tests__/__snapshots__/demo-extend.test.ts.snap:12719]: 重新装下依赖，这个是 next 的结构

**Comment by MadCcc** [comment]
感觉可以直接写在 changelog 里

File: components/space/__tests__/__snapshots__/demo-extend.test.ts.snap (hunk 2 above)
Comment (li-jia-nan) [reply components/space/__tests__/__snapshots__/demo-extend.test.ts.snap:12719]: 怪不得，我重装试试

**Comment by li-jia-nan** [comment]
> 感觉可以直接写在 changelog 里

文档里面也写一下吧，这样对用户友好一点，并不是所有的用户都会看 changelog

**Pull request merged**